moveit_ros/moveit_servo/launch/cpp_interface_example.launch

```diff
@@ -2,8 +2,8 @@
   <!-- Launch an example that sends commands via C++ API. -->
 
   <node name="servo_server" pkg="moveit_servo" type="cpp_interface_example" output="screen" >
-    <param name="parameter_ns" type="string" value="servo_server" />
-    <rosparam command="load" file="$(find moveit_servo)/config/ur_simulated_config.yaml" />
+    <param name="parameter_ns" type="string" value="optional_parameter_namespace" />
```

Comment:
Just curious, what exactly is the purpose of this optional subnamespace?

Reply:
Suppose you had a node named `foobar_node`. Normally parameters would be loaded into `/foobar_node/<parameters>`. If the node already has many parameters maybe you want to make things a bit clearer by having `foobar_node` parameters inside namespace `/foobar_node/<parameters>` and all the servo parameters (there are quite a few of them) within subnamespace `/foobar_node/servo_params/<parameters>`. You can also stash multiple servo configurations within single config file and place them into named sections and use that subnamespace to access them.

Reply:
I'm OK with this sub-namespace approach. Seems like a good idea